TLCGen.PluggedInItems/TLCGen.Generators.CCOL/CodeGeneration/Sources/CCOLGeneratorGenerateRegC.cs

```diff
@@ -595,11 +595,19 @@ private string GenerateRegCFileVerwerking(ControllerModel controller)
 
             sb.AppendLine("void FileVerwerking(void)");
             sb.AppendLine("{");
-            sb.AppendLine($"{ts}#if !defined CUSTOM_FILEVERWERKING");
-
+            if (controller.FileIngrepen.Any())
```

Comment:
deze check lijkt me niet nodig? De functie waarbinnen deze code wordt aangemaakt zit al een check op naar ik meen

Reply:
Naar mijn smaak is het niet fraai om wanneer er geen file ingrepen zijn geconfigureerd, in de reg.c de @ifndef code op te nemen die gebruikt kan worden wanneer er wel fileingrepen zouden zijn geconfigureerd. Daarom gemaakt dat de #infndef alleen wordt gegenereerd bij daadwerkelijke file ingrepen.
Maar dit is geen halszaak.

Reply:
Volgens mij wordt de hele fileverwerking functie niet aangemaakt als je geen file ingrepen hebt. Maar, ok om dit even te laten staan. Ik zou het evenwel *om* de `define` doen. Maargoed, maar zo laten nu.